### DISCUSSION TITLE
Feature request: Per-user installation

### DISCUSSION THREAD

**Discussion by win98se** [valinet/ExplorerPatcher#685] [Ideas]
Although the current approach is using the DLL method (as I have read [here](https://github.com/valinet/ExplorerPatcher/issues/642#issuecomment-1007880758)), is there any possibility that per-user installation is supported?

**Comment by valinet** [comment]
That comment is from 10 days ago or so, during which I made 4 commits or so to this, none relating to this matter. So, the answer still stands. Installation requires files to be placed in system directories. Even if implemented differently, it would likely still need this to achieve full functionality, so it better do it from the get go. Also, I don't see the scenario for this, honestly. Anyway, after you install the software, its configuration is available for each user to customize; thus, you could have the Windows 10 taskbar enabled only on your account, and the Windows 11 UI on the others, although the DLL would still be loaded in both instances. Also, beware that this is not really tested in multi-user scenarios.
